keyless/pepper/service/src/lib.rs

```diff
@@ -248,7 +248,7 @@ impl HandlerTrait<VerifyRequest, VerifyResponse> for V0VerifyHandler {
                             }
 
                             let result = zksig
-                                .verify_groth16_proof(public_inputs_hash, &DEVNET_VERIFICATION_KEY);
+                                .verify_groth16_proof(public_inputs_hash, &VERIFICATION_KEY_FOR_TESTING);
```

Comment:
@zjma and @heliuchuan this don't look right: why is the pepper service using a hardcoded VK? It should fetch it from the chain. (I think this is in relation to [this PR](https://github.com/aptos-labs/aptos-core/pull/14344), which is not documented in the AIP and should be; will create a task).

Reply:
Yes this should be fetched from the chain